app/oauth/consent.js

```diff
@@ -12,9 +12,9 @@ async function idTokenExtraParams (sub) {
   const conn = await db()
   const [user] = await conn('users').where('id', sub)
   const { profile } = user
-  const displayName = profile.displayName || consent.subject
```

Comment:
`consent` is undefined here so presumably this never worked and it's fine to remove it?

Reply:
the comment on the diff makes it look almost like the whole line is removed but its just ` || consent.subject` that is removed.

Reply:
It should be the argument to the function, `sub`